yt_dlp/extractor/yourreformer.py

```diff
@@ -0,0 +1,85 @@
+from .common import InfoExtractor
+
+
+class YourReformerIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?yourreformerathome\.uscreen\.io/programs/(?P<id>[^/?#&]+)'
+    _TESTS = [{
+        'url': 'https://yourreformerathome.uscreen.io/programs/arms-back-abs-7-with-erin?category_id=139586',
+        'info_dict': {
+            'id': 'arms-back-abs-7-with-erin',
+            'ext': 'mp4',
+            'title': 'Arms, Back & Abs 7',
+        },
+        'params': {
+            'skip_download': True,
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        # Extract category_id from the URL
+        category_id = None
+        if '?' in url:
```

Comment:
use parse_qs from utils for parsing url queries (with maybe traverse_obj - should be some examples in the codebase)

Reply:
Turns out this request straight up doesn't need the category and can be stripped entirely.